packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -326,9 +326,6 @@ class TextTreeConfiguration {
 
   /// Mandatory string to add after the properties of a node regardless of
   /// whether the node has any properties.
-  ///
-  /// See [headerLineTextConfiguration] for an example of using this field to
```

Comment:
`singleLineTextConfiguration`?

Reply:
That field sets this to `''` - doesn't seem like a great example :\